### ISSUE TITLE
[TASK] Update extension and make it compatible with TYPO3 7.x

### ISSUE THREAD

**Issue by soee** [bootstraptheme-for-typo3/fluidbootstraptheme#273]
It's been a while since we have TYPO3 7.x available and this extensions still sets max allowed version to 6.2.x. We should release updated version compatible with 7.x as soon as possible.

This extesnions depends on:
_flux_ - current TER version TYPO3 compatibility: _6.2.0 - 7.1.99_
_fluidcontent_ - current TER version TYPO3 compatibility: _6.1.0 - 7.1.99_
_fluidpages_ - current TER version TYPO3 compatibility: _6.2.0 - 7.1.99_
_vhs_ - current TER version TYPO3 compatibility: _6.2.0 - 7.99.99_

so ... this extensions should be at least updated to support TYPO3 6.1.0 - 7.1.99

**Comment by misterboe** [comment]
for me flux, fluidcontent and fluidpages is not working in 7.x ..

**Comment by soee** [comment]
@misterboe what are the problems with them and why they are marked as compatible than?

**Comment by xf-** [comment]
hi,

we updated a lot in last days. Biggest issue should be a core issue with container.
7.3.1 should be a good base.

Please report all bugs to us and ignore the set working versions.

**Comment by xf-** [comment]
PR is open
https://github.com/bootstraptheme-for-typo3/fluidbootstraptheme/pull/274

**Comment by farconada** [comment]
Acordion and Tabs dont work with v7.x

**Comment by xf-** [comment]
I added the commit to set current versions to avoid the Message about compatibly.
Most stuff works :)

**Comment by farconada** [comment]
yeah, I tried it but accordion and tabs dont work for example, all other content elements seem to work well

**Comment by xf-** [comment]
@farconada https://github.com/bootstraptheme-for-typo3/fluidbootstraptheme/pull/275 should fix the issue

**Comment by fabarea** [comment]
It looks we can go forward with this task now that we have all major FluidTYPO3 extensions compatible with TYPO3 7 LTS.

**Comment by outdoorsman** [comment]
So what's the state on the 7 LTS compatibility now?

**Comment by fschaeffer** [comment]
may I also ask whether there is any progress in making this extension 7 LTS compatible? what is missing?
